src/bootstrap/Cargo.toml

```diff
@@ -34,8 +34,8 @@ build_helper = { path = "../tools/build_helper" }
 cmake = "0.1.38"
 fd-lock = "3.0.8"
 filetime = "0.2"
-getopts = "0.2.19"
 cc = "1.0.69"
+clap = { version = "4.1.4", features = ["derive"] }
```

Comment:
You can try `default-features = false` and turn on only some of them to reduce build time.

Reply:
Looks like it saves a few seconds, thanks